src/components/PhotoUpload/PhotoUploadButton.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+
+import styled from 'styled-components';
+import * as React from 'react';
+import colors from '../../lib/colors';
+import { t } from 'c-3po';
+
+import CameraIcon from './CameraIcon';
+
+type Props = {
+  className: string,
+  onClick?: ((event: UIEvent) => void),
+};
+
+type State = {
+  sourceName?: string;
+};
+
+const defaultState: State = {};
+
+const StyledCameraIcon = styled(CameraIcon)`
+  font-size: 2em;
+
+`;
+
+class PhotoUploadButton extends React.Component<Props, State> {
+  props: Props;
+  state: State = defaultState;
+
+  componentDidMount() {
+  }
+
+  componentWillReceiveProps(newProps: Props) {
+  }
```

Comment:
Do we still need these method stubs?

Reply:
Na, see above, will go away at some point.